index.bs

```diff
@@ -778,6 +778,41 @@ to display an overlay which informs the user that they have entered fullscreen,
 and to advertise a simple mechanism to exit fullscreen (typically the `Esc` key).
 </div>
 
+<h3 class=question id="non-fully-active">
+  How does your feature handle non-"fully active" documents?
+</h3>
+
+After a user navigated away from a document,
+the document might stay around in a non-"[=Document/fully active=]" state,
+and might be reused when the user navigates back to the entry holding the document.
```

Comment:
not sure "to the entry" is easy to understand. Maybe just remove?

Reply:
Removed, thanks!